### PR TITLE
Scb 180

### PR DESCRIPTION
**Pull request by mattkimtx**

### CODE REVIEW

File: myenv/lib/python3.12/site-packages/Jinja2-3.1.2.dist-info/LICENSE.rst
Comment (Anita-TZ) [inline myenv/lib/python3.12/site-packages/Jinja2-3.1.2.dist-info/LICENSE.rst:1]: Hi Jino,

Normally, we would not upload the python package.
If other users are willing to operate the project, they might get the necessary package by installing requirement.txt.

File: requirements.txt [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
 cachetools==5.3.2
 certifi==2023.11.17
 cffi==1.16.0
 charset-normalizer==3.3.2
+click==8.1.7
 cryptography==41.0.7
 Django==5.0
+djangorestframework==3.14.0
 dnspython==2.4.2
+Flask==3.0.0
```
Comment (Anita-TZ) [inline requirements.txt]: just curious.

Why do we need to use Flask here?

File: myenv/bin/flask
Comment (Anita-TZ) [inline myenv/bin/flask:1]: The whole myenv folder might not be necessary to be uploaded. users might install suitable environment by installing requirement.txt.

Try uploading only the minimum necessary files, and clear instructions for running the project.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+COPY . /senao-intern-project
+
+# Install any Python dependencies
+# RUN pip install -r requirements.txt
+
+# Expose the port that your Django application will run on (e.g., 8000)
+EXPOSE 8000
+
+# Start the Django development server
+CMD ["/bin/bash", "-c", "source ./myenv/bin/activate && pip install -r requirements.txt && cd mysite && python manage.py runserver 0.0.0.0:8000"]
```
Comment (Anita-TZ) [inline Dockerfile]: What docker creates is like a virtual environment, and its purpose is equivalent to what my env folder does in order to creating a independent but suitable env for project running.
Therefore, when using docker, normally there is no need to activate the myenv virtual environment.

File: mysite/account/__pycache__/__init__.cpython-312.pyc
Comment (Anita-TZ) [inline mysite/account/__pycache__/__init__.cpython-312.pyc:1]: The __pycache__ folder is automatically created by Python to store compiled bytecode files (.pyc) that result from the interpretation of Python source files (.py). 

and it is not necessary to be uploaded for running the project, either.

maybe try to use .gitignore to help prevent unnecessary files from uploading to github.

File: mysite/.DS_Store
Comment (Anita-TZ) [inline mysite/.DS_Store:1]: what is the purpose of the file?

File: requirements.txt (hunk 1 above)
Comment (mattkimtx) [reply requirements.txt]: Oh sorry! Originally I was going to use flask for part of my user authentication, but I ended up not using it. I will remove!

File: mysite/.DS_Store
Comment (mattkimtx) [reply mysite/.DS_Store:1]: Oh this file gets uploaded when doing github push from a mac. Sorry, I forgot to do git ignore for this. I'm not sure why apple makes each directory have a .DS_Store.

**Pull request merged**